src/controller/expense.controller.js

```diff
@@ -0,0 +1,74 @@
+const expenseServise = require('../services/expenseService');
+const usersService = require('../services/userService');
+
+const expenseController = {};
+
+expenseController.get = (req, res) => {
```

Comment:
```suggestion
const getAll = (req, res) => {
```

The same with all other functions